scripts/aguythatcodes.js

```diff
@@ -0,0 +1,4 @@
+let fullname = "Emmanuel Amodu";
+let id = "HNG-05308";
+let language = "Javascript";
+console.log(`Hello World, this is ${fullname} with HNGi7 ID ${id} using ${language} for stage 2 task`);
```

Comment:
No email

Reply:
I dont understand are we to add email to the text to be logged in the script